components/src/preact/mutationsOverTime/mutations-over-time.tsx

```diff
@@ -154,13 +155,16 @@ const MutationsOverTimeTabs: FunctionComponent<MutationOverTimeTabsProps> = ({
         { label: 'Deletions', checked: true, type: 'deletion' },
     ]);
 
+    const [gapsHidden, setGapsHidden] = useState<boolean>(originalComponentProps.initialGapsHidden ?? false);
```

Comment:
I think it would be less surprising if this was controllable from the outside. It's not likely that users will change it from the outside, but I think it would be good to add it.

That should actually do it:
```suggestion
    const [gapsHidden, setGapsHidden] = useState<boolean>(originalComponentProps.initialGapsHidden ?? false);
    
    useEffect(
        () => setGapsHidden(originalComponentProps.initialGapsHidden),
        [originalComponentProps.initialGapsHidden],
    )
```